cmake/DependenciesAndroid.cmake

```diff
@@ -0,0 +1,28 @@
+macro(add_android_dependencies)
+
+	find_package(SDL2 REQUIRED)
+
+    add_library(SDL_1_XX INTERFACE)
+    target_link_libraries(SDL_1_XX INTERFACE
+		SDL2::SDL2-static
+		SDL2::SDL2main
+    )
+
+    add_library(Dependencies INTERFACE)
+    target_link_libraries(Dependencies INTERFACE
+            -lGLESv1_CM
+            -landroid
+            -llog
+            -ldl
+            -lz
+            -lEGL
+    )
+
+    target_compile_definitions(Dependencies INTERFACE
+            SPELUNKY_PSP_PLATFORM_ANDROID
+    )
+endmacro()
+
+macro(spelunky_psp_post_build_android)
+
```

Comment:
Calling `gradlew assemble` here as a post-build action could be a good place for it.

Reply:
After re-thinking this, I am choosing to call `gradle` separately, with Spelunky_PSP as `externalNativeBuild` CMake target.